healme.lic

```diff
@@ -35,15 +35,19 @@ class HealMe
     wounds.map { |wound| wound['level'] }.inject(&:+)
   end
 
-  def spam_heal(base, camb)
+  def spam_heal(base, camb, spells)
```

Comment:
❓ What do you think about passing in `spells['Heal']` here (instead of `spells`)?

Reply:
we can't without eliminating the empath_healing integration from yaml